test/github-issues/3256/issue-3256.ts

```diff
@@ -2,13 +2,15 @@ import "reflect-metadata";
 import {closeTestingConnections, createTestingConnections, reloadTestingDatabases} from "../../utils/test-utils";
 import {Connection} from "../../../src/connection/Connection";
 import {Post} from "./entity/Post";
+import { OldEntityFactory } from "../../../src";
 
 describe("github issues > #3256 wrong subscriber methods being called", () => {
 
     let connections: Connection[];
     before(async () => connections = await createTestingConnections({
         entities: [__dirname + "/entity/*{.js,.ts}"],
         subscribers: [__dirname + "/subscriber/*{.js,.ts}"],
+        entityFactory:  new OldEntityFactory()
```

Comment:
why do we need it now? Are those tests failing without old entity factory now?

Reply:
As above - fields are initialized. To make it work without OldEntityFactory we would have to change them to check for undefined values.
https://github.com/typeorm/typeorm/blob/d3ee978501a4f1b74c99e2fc24ed5d8105919322/test/github-issues/3256/issue-3256.ts#L32